patient/README.md

```diff
@@ -6,9 +6,13 @@
 {"v":1,"t":1234,"P":12.332,"F":-1.23,"temp":23.3}
 ```
 
-Note that this is 1 per line, produced 50 times per second.
-Order does not matter in JSON. `"v":1` is for version 1 - we could later
-add a new version.
+One [JSON object](https://www.json.org/json-en.html) per line of text (delimited by `"\n"`, no `"\n"` allowed within a JSON object) at a rate of roughly 50 Hz. Field names are case-sensitive.
+
+   * The `"v"` field (mandatory) is the version of the protocol; always `1`. Future versions are also integers (and version updates will be rare).
+   * The `"t"` field (mandatory) is a timestamp in ISO 8601 format (string) with milliseconds.
```

Comment:
Will the patient π's know the current time? They are unlikely to be ever connected to a network other than the nurse station. I was thinking that this would be some arbitrary number, and then the nurse station would try to assign realistic times to it based on when the packets were received. Missing the time by a second or two is just fine in this case.